src/CovidAppointmentTable.js

```diff
@@ -151,7 +153,21 @@ export default function CovidAppointmentTable() {
                 />
                 {ready && formattedData.length === 0 && (
                     <div role="status">
-                        <p>No appointments found.</p>
+                        <br/>
+                        <Alert severity={"info"}>
+                            <AlertTitle >
+                                No Appointments Found
+                            </AlertTitle>
+                                <p>
+                                    This website scrapes data from other websites every few minutes and tells you
+                                    what places currently are advertising available appointments for COVID vaccines.
+                                </p>
+                                <p>
+                                    Mass vaccination locations have the most appointments and update their schedules every Thursday.
```

Comment:
As noted in the Issue (#72), this claim about Thursdays appears to be false and unadvisible.
The first half of the sentence seems ok tho?

Reply:
Sorry.  I accidentally made the "pull request" by hitting some random key on my keyboard.  I'm going through the comments in the issue.  Thanks!